src/components/biggive-search/biggive-search.scss

```diff
@@ -0,0 +1,120 @@
+.container {
+  @include standard-font();
+  @include all-round-shadow();
+  width: 400px;
+  padding: 1.5rem;
+
+  h2 {
+    @include standard-font();
+    width: 100%;
+    font-weight: 500;
+    margin-top: 0px;
+    margin-bottom: 0px;
+    text-transform: uppercase;
+    font-size: 1.4rem;
+  }
+
+  .search-box {
+    display: flex;
+    background-color: #f1f1f1;
+    padding: 15px 10px;
+    margin-top: 1rem;
+    margin-bottom: 1.25rem;
+    border-radius: 5px;
+
+    .icon {
+      @include icon();
+      filter: opacity(0.3);
+      margin: auto;
+    }
+  
+    input {
+      width: 100%;
+      box-sizing: border-box;
+      border: none;
+      background-color: #f1f1f1;
+      border-radius: 3px;
+      font-size: 1.1rem;
+  
+      &::placeholder {
+        color: rgb(150, 150, 150);
+      }
+  
+      &:focus {
+        outline: none;
+      }
+    }
+  }
+
+  button {
+    @include standard-font();
+    width: 100%;
+    padding: 1.5rem 0.75rem;
+    box-sizing: border-box;
+    background-color: $colour-primary;
+    color: white;
+    border: none;
+    border-radius: 5px;
+    text-transform: uppercase;
+    font-size: 1rem;
+
+    &:hover {
+      cursor: pointer;
+      background-color: navy;
+    }
+  }
+}
+
+@media screen and (max-width: 1024px) {
+    .container {
+      width: 350px;
+
+      h2 {
+        font-size: 1.3rem;
+      }
+
+      input {
+        font-size: 0.9rem;
+      }
+    }
+}
+
+@media screen and (max-width: 768px) {
+  .container {
+    width: 275px;
+
+    h2 {
+      font-size: 1.2rem;
+    }
+  }
+}
+
+@media screen and (max-width: 425px) {
+  .container {
+    width: 100vw;
+    display: table-cell; // so the padding doesn't mess up
```

Comment:
How did it mess up? It seems very risky / potentially brittle to have the entire layout model change depending on your screen size!